### ISSUE TITLE
App Crash when use the GCSVideoView in the storyboard

### ISSUE THREAD

**Issue by congsun** [googlesamples/cardboard-ios#5]
When I alloc init it by code and addsubview, it seems working. However, if I assign the GCSVideoView in the storyboard to a UIView and use IBOutlet, the app will crash at launch. Maybe something wrong in the initWithCoder?
```
2016-04-04 14:55:54.400 Google360Video[1897:18511] *** Assertion failure in -[GCSVideoView onMetadataParsed:], googlemac/iPhone/CardboardSDK/Source/GCSVideoView.mm:227
2016-04-04 14:55:54.407 Google360Video[1897:18511] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Error at CVOpenGLESTextureCacheCreate -6661'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010ab65d85 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x000000010a5d9deb objc_exception_throw + 48
	2   CoreFoundation                      0x000000010ab65bea +[NSException raise:format:arguments:] + 106
	3   Foundation                          0x000000010a223d5a -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 198
	4   Google360Video                      0x00000001084662d2 -[GCSVideoView onMetadataParsed:] + 838
	5   Google360Video                      0x0000000108465c3e -[GCSVideoView connection:didReceiveData:] + 421
	6   CFNetwork                           0x000000010e534a36 __65-[NSURLConnectionInternal _withConnectionAndDelegate:onlyActive:]_block_invoke + 69
	7   CFNetwork                           0x000000010e5349da -[NSURLConnectionInternal _withConnectionAndDelegate:onlyActive:] + 199
	8   CFNetwork                           0x000000010e534b2f -[NSURLConnectionInternal _withActiveConnectionAndDelegate:] + 48
	9   CFNetwork                           0x000000010e509166 _ZL30_NSURLConnectionDidReceiveDataP16_CFURLConnectionPK8__CFDatalPKv + 79
	10  CFNetwork                           0x000000010e399655 ___ZN27URLConnectionClient_Classic29_delegate_didReceiveDataArrayEv_block_invoke + 293
	11  CFNetwork                           0x000000010e48fd75 ___ZN27URLConnectionClient_Classic18_withDelegateAsyncEPKcU13block_pointerFvP16_CFURLConnectionPK33CFURLConnectionClientCurrent_VMaxE_block_invoke_2 + 100
	12  libdispatch.dylib                   0x000000010c8613eb _dispatch_client_callout + 8
	13  libdispatch.dylib                   0x000000010c84348b _dispatch_block_invoke + 716
	14  CFNetwork                           0x000000010e38860a _ZN19RunloopBlockContext13_invoke_blockEPKvPv + 24
	15  CoreFoundation                      0x000000010aa5e894 CFArrayApplyFunction + 68
	16  CFNetwork                           0x000000010e388503 _ZN19RunloopBlockContext7performEv + 137
	17  CFNetwork                           0x000000010e3883bc _ZN17MultiplexerSource7performEv + 282
	18  CFNetwork                           0x000000010e3881de _ZN17MultiplexerSource8_performEPv + 72
	19  CoreFoundation                      0x000000010aa8b301 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	20  CoreFoundation                      0x000000010aa8122c __CFRunLoopDoSources0 + 556
	21  CoreFoundation                      0x000000010aa806e3 __CFRunLoopRun + 867
	22  CoreFoundation                      0x000000010aa800f8 CFRunLoopRunSpecific + 488
	23  GraphicsServices                    0x000000010ddaaad2 GSEventRunModal + 161
	24  UIKit                               0x000000010b1c1f09 UIApplicationMain + 171
	25  Google360Video                      0x000000010844f06f main + 111
	26  libdyld.dylib                       0x000000010c89592d start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by nathanmartz** [comment]
Hi. We've confirmed this bug. Fix will be in our next major update. Until then, please assign the GDCVideoView programatically rather than in the storyboard.

**Comment by sanjayc77** [comment]
Fixed in 0.7.2
